mpp/shmem.h.in

```diff
@@ -18,6 +18,10 @@
 
 #include <stddef.h>
 #include <complex.h>
```

Comment:
Should this be included for C++?  This seems like it will cause an issue with the ```#define``` below?